zenpy/lib/request.py

```diff
@@ -118,6 +118,33 @@ def delete(self,
         return response
 
 
+class ArticleCRUDRequest(CRUDRequest):
+    def post(self, api_objects, *args, **kwargs):
```

Comment:
nitpick: there is a fair bit of repeated code (cf `CRUDRequest`) here.  There might be some way to DRY it up a tad.

Reply:
will dry